test_common/gl/setup_x11.cpp

```diff
@@ -13,15 +13,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
-#define GL_GLEXT_PROTOTYPES
```

Comment:
This change is good.

It looks setup_win32.cpp defines GL_GLEXT_PROTOTYPES also, even though win32 isn't using glext.h anymore either.  Could we clean it up too?

Reply:
removed in https://github.com/KhronosGroup/OpenCL-CTS/pull/1337/commits/4db06f50ee26d58454edceaebb6c076b25b718b0